src/html/index.ejs

```diff
@@ -93,7 +93,7 @@
 
 
 		<footer class="text-center bg-light text-muted">
-			<%= htmlWebpackPlugin.options.title || 'Osweb'%> — Copyright Jaap Bos, Daniël Schreij & Sebastiaan Mathôt 2016-2017<br>
+			<%= htmlWebpackPlugin.options.title || 'Osweb'%> — Copyright Jaap Bos, Daniël Schreij & Sebastiaan Mathôt 2016-2018<br>
```

Comment:
Wouldn't it be better to determine the current year automatically? Simply adding a `new Date().getFullYear()` should do the trick here.

Reply:
Yes, even better!